modules/openy_features/openy_location/modules/openy_loc_filter/openy_loc_filter.libraries.yml

```diff
@@ -0,0 +1,9 @@
+openy_location_filter:
+  version: VERSION
+  js:
+    js/location_filter.js: {}
```

Comment:
is it minified?
Can we minify it and add a minified: true in options?
Let's use webpack or bundler for automation here

Reply:
Fixed

Reply:
MInifying a js file with only one small Drupal behavior that will be included and used only in the admin interface looks redundant. 
Fixing this we added many more files into the PR and difficulties to support this simple module. 
@podarok I suggest removing modification and leave at as is.